apiv2/src/admin/core/referentiel/classe/useCase/ImporterClasses.ts

```diff
@@ -186,6 +187,20 @@ export class ImporterClasses implements UseCase<ClasseRapport[]> {
             originalSessionId: jeune.sessionId,
             originalSessionNom: jeune.sessionNom,
             sessionChangeReason: "Import SI-SNU",
+            statutPhase1: jeune.statutPhase1 === "AFFECTED" ? "WAITING_AFFECTATION" : jeune.statutPhase1,
+            ligneDeBusId: undefined,
+            meetingPointId: undefined,
+            centreId: undefined,
+            pointDeRassemblementId: undefined,
+            hasMeetingInformation: undefined,
+            transportInfoGivenByLocal: undefined,
+            deplacementPhase1Autonomous: undefined,
+            cohesionStayPresence: undefined,
+            presenceJDM: undefined,
+            departInform: undefined,
+            departSejourAt: undefined,
+            departSejourMotif: undefined,
+            departSejourMotifComment: undefined,
```

Comment:
```suggestion
            centreId: undefined,
            sejourId: undefined,
            pointDeRassemblementId: undefined,
            ligneDeBusId: undefined,
            hasPDR: undefined,
            transportInfoGivenByLocal: undefined,
            deplacementPhase1Autonomous: undefined,
            presenceArrivee: undefined,
            presenceJDM: undefined,
            departInform: undefined,
            departSejourAt: undefined,
            departSejourMotif: undefined,
            departSejourMotifComment: undefined,
            youngPhase1Agreement: "false",
```
cf `ValiderBasculeJeunes.service.ts` (basculerJeune) ou `Desistement.service.ts` (desisterJeunes)